package/MDAnalysis/coordinates/chain.py

```diff
@@ -656,3 +643,17 @@ def _apply_transformations(self, ts):
         # to avoid applying the same transformations multiple times on each frame
 
         return ts
+
+    def __next__(self):
+        if self._current_frame < self.n_frames - 1:
+            j, f = self._get_local_frame(self._current_frame + 1)
+            self.__activate_reader(j)
+            self.ts = self.active_reader[f]
+            self.ts.frame = self._current_frame + 1
+            self._current_frame += 1
+            return self.ts
+        else:
+            raise StopIteration()
+
+    def next(self):
+        return self.__next__()
```

Comment:
The `next` method is not needed anymore. See [pep-3114](https://www.python.org/dev/peps/pep-3114/). I know we are not very good in following this pattern.

Reply:
If we backport this fix to 1.0.1 #2798 , will we need to add `next()` back for Python 2.7?

Reply:
`next()` still functions here from `ProtoReader` definition.

Reply:
Than we should leave the next function and remove it once from all readers.